static-site/content/docs/02-bioinformatics/05-data-formats.md

```diff
@@ -43,3 +43,7 @@ The filename _must_ end with `_meta.json` and have the same prefix as the tree J
 
 Currently this is only used by the flu builds, and generates the frequencies panel you can see at [nextstrain.org/flu](https://nextstrain.org/flu). [Here](http://data.nextstrain.org/flu_seasonal_h3n2_ha_2y_tip-frequencies.json) is an example of this file.
 
+## Input files for Auspice
+_Full information on file format requirements for Auspice and how to get your data into the correct format will be updated soon_
+
+If you are interested in visualizing Newick trees, which is possible with CSV/TSV metadata files, then you may want to use [auspice.us](https://auspice-us.herokuapp.com/), which lets you drag and frop your files into the browser.
```

Comment:
If you'd like to promote auspice.us then please add a caveat that it's in a very early prototype phase at the moment. 

P.S. `/frop/drop/`

Reply:
Added this as it was something that was in the auspice v2 docs that I thought might be useful to have in the nextstrain.org docs. Happy to remove it especially if it's still very much a prototype. Pushed a commit with that language removed.